cmd/deployment/update_help.go

```diff
@@ -0,0 +1,116 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmddeployment
+
+const updateLong = `updates a deployment from a file definition, defaulting prune_orphans=false, making the default
+update action safe for partial updates, to override this behavior toggle --prune-orphans.
+To track the changes toggle the --track flag.
+
+It is possible to use "--generate-payload" as: "ecctl deployment show --generate-payload ..."
+to obtain a valid create / update payload from an existing deployment which can be manually edited in
```

Comment:
```suggestion
to obtain a valid update payload from an existing deployment which can be manually edited in
```
same as above, can we double check?